lib/core-integration/src/Test/Integration/Framework/TestData.hs

```diff
@@ -344,6 +346,14 @@ errMsg404NoEndpoint = "I couldn't find the requested endpoint. If the endpoint\
     \ contains path parameters, please ensure they are well-formed, otherwise I\
     \ won't be able to route them correctly."
 
+errMsg404NoPendingAnymore :: Text -> String
```

Comment:
I think this error code should be something 4xx, but not 404.

Reply:
Good point. `403` would be a better choice here:

> The request contained valid data and was understood by the server, but the server is refusing action. This may be due to the user not having the necessary permissions for a resource or needing an account of some sort, or attempting a prohibited action